config/skim_Legacy2018.cfg

```diff
@@ -98,60 +98,61 @@ skipTriggers = false
 
 
 [BDTmm]
 computeMVA = false
-weights    = /home/llr/cms/amendola/HHLegacy/CMSSW_11_1_0_pre6/src/KLUBAnalysis/weights/newBDT/BDTnewMM.xml
+weights    = /home/llr/cms/motta/HHLegacy/CMSSW_11_1_0_pre6/src/KLUBAnalysis/weights/newBDT/BDTnewMM.xml
 mass       = 250, 280, 320, 350, 400, 450, 650, 900
 spin       = 0 #, 1
 variables  = tauH_pt:pt_l1l2, p_zeta:p_zeta, BDT_dib_abs_deltaPhi:abs_dphi_b1b2, BDT_tauHsvfitMet_abs_deltaPhi:abs_dphi_METhtautau_sv, BDT_HHsvfit_abs_deltaPhi:abs_dphi_hbbhatutau_sv, dib_deltaEta:abs_deta_b1b2, dau2_MET_deltaEta:abs_deta_l2MET, bH_MET_deltaEta:abs_deta_hbbMET, ditau_deltaR:dR_l1l2, mT1:MT_l1, mT_tauH_SVFIT_MET:MT_htautau_sv, mT_total:MT_tot, BDT_MX:mass_X, BDT_bH_tauH_InvMass:mass_H_vis,  HHKin_mass_raw:mass_H_kinfit, HHKin_mass_raw_chi2:mass_H_kinfit_chi2, BDT_total_CalcPhi:phi_sv, BDT_ditau_CalcPhi:phi_1_sv, BDT_b1_bH_cosTheta:costheta_b1hbb, BDT_tauH_SVFIT_reson_cosTheta:costheta_htautau_svhhMET
 
 
 [BDThm]
 computeMVA = false
-weights    = /home/llr/cms/amendola/HHLegacy/CMSSW_11_1_0_pre6/src/KLUBAnalysis/weights/newBDT/BDTnewHM.xml
+weights    = /home/llr/cms/motta/HHLegacy/CMSSW_11_1_0_pre6/src/KLUBAnalysis/weights/newBDT/BDTnewHM.xml
 mass       = 250, 280, 320, 350, 400, 450, 650, 900
 spin       = 0 #, 1
 variables  = bH_pt:pt_hbb, dau1_pt:pt_l1, dau2_pt:pt_l2, tauH_SVFIT_pt:pt_htautau_sv, BDT_HT20:HT_otherjets, p_zeta:p_zeta, p_zeta_visible:p_zetavisible, BDT_tauHsvfitMet_abs_deltaPhi:abs_dphi_METhtautau_sv, BDT_bHMet_deltaPhi:dphi_hbbMET, ditau_deltaR:dR_l1l2, bH_tauH_MET_deltaR:dR_hbbhtautau, tauH_SVFIT_mass:mass_htautau_sv, mT_tauH_MET:MT_htautau, MT2:MT2, BDT_MX:mass_X,  BDT_bH_tauH_MET_InvMass:mass_H, BDT_bH_tauH_SVFIT_InvMass:mass_H_sv, BDT_bH_tauH_InvMass:mass_H_vis, HHKin_mass_raw:mass_H_kinfit, HHKin_mass_raw_chi2:mass_H_kinfit_chi2
 
 
 [BDTVBF]
 computeMVA = false
-weights    = /home/llr/cms/amendola/HHLegacy/CMSSW_11_1_0_pre6/src/KLUBAnalysis/weights/newBDT/BDTVBF.xml
+weights    = /home/llr/cms/motta/HHLegacy/CMSSW_11_1_0_pre6/src/KLUBAnalysis/weights/newBDT/BDTVBF.xml
 #variables  = VBFjj_dEtaSign:BDT_j1j2n_dEtaSign, HH_deltaR:BDT_n_HH_deltaR, dib_dEtaSign:BDT_n_b1b2_dEtaSign, VBFjj_deltaEta:BDT_j1j2n_dEta, VBFjj_mass:BDT_j1j2n_Mjj, VBFjet2_pt:BDT_j2n_pt, tauH_pt:BDT_n_ditau_pt, bH_VBF1_deltaEta:BDT_bbH_j1n_dEta, VBFjet1_btag:BDT_j1n_csv, VBFjet1_pt:BDT_j1n_pt, dib_deltaR:BDT_n_dib_deltaR, VBFjet2_PUjetID:BDT_j2n_PUjetID
 variables  = VBFjet1_pt:BDT_j1n_pt, bH_pt:BDT_n_bH_pt, dib_dEtaSign:BDT_n_b1b2_dEtaSign, ditau_deltaR:ditau_deltaR, HH_deltaR:BDT_n_HH_deltaR, HH_zV:BDT_HH_zV_n, VBFjj_dEtaSign:BDT_j1j2n_dEtaSign, VBFjj_mass:BDT_j1j2n_Mjj
 
 
 [DYLOtoNLOreweight]
 doReweight = true
```

Comment:
I do not know if this doReweight does something but I think it should be false right?